### PR TITLE
Feat: add scope label for workflowrun defs

### PR DESCRIPTION
**Pull request by FogDong**
Signed-off-by: FogDong <dongtianxin.tx@alibaba-inc.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

####  Verified Addon promotion rules

If this pr wants to promote an experimental addon to verified, you must check whether meet these conditions too:
  - [ ] This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can be enabled successfully.
  - This addon must have some basic but necessary information.
    - [ ] An accessible icon url and source url defined in addon's `metadata.yaml`.
    - [ ] A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
    - [ ] Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
    - [ ] It's more likely to be accepted if useful examples provided in example [dir](examples/).

### CODE REVIEW

File: addons/vela-workflow/definitions/addon-operation.cue [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	}
 	labels: {
 		"ui-hidden": "true"
+		"origin":    "WorkflowRun"
```
Comment (wonderflow) [inline addons/vela-workflow/definitions/addon-operation.cue:12]:
```suggestion
		"created-by":    "WorkflowRun"
```

origin is confused

**Review by wangyikewxgm — APPROVED** [review]

File: addons/vela-workflow/definitions/addon-operation.cue [hunk 2]
```diff
@@ -9,6 +9,7 @@ import (
 	}
 	labels: {
 		"ui-hidden": "true"
```
Comment (barnettZQG) [inline addons/vela-workflow/definitions/addon-operation.cue:11]: Remove the 'ui-hidden' label.

File: addons/vela-workflow/definitions/clean-jobs.cue [hunk 3]
```diff
@@ -7,6 +7,7 @@ import (
 	annotations: {}
 	labels: {
 		"ui-hidden": "true"
```
Comment (barnettZQG) [inline addons/vela-workflow/definitions/clean-jobs.cue:9]: Same as above.

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**